daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -846,6 +846,29 @@ class RazerOrnataV3Tenkeyless(_RippleKeyboard):
     DEVICE_IMAGE = "https://dl.razerzone.com/src2/13038/13038-1-en-v1.png"
 
 
+class RazerBlackWidowV4_75PCT(_RippleKeyboard):
+    """
+    Class for the Razer BlackWidow V4 75%
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_BlackWidow_V4_75PCT(-if01)?-event-kbd')
```

Comment:
This seems wrong. #2123 has `/dev/input/by-id/usb-Razer_Razer_BlackWidow_V4_75_-event-kbd` so the regex should become `r'.*Razer_BlackWidow_V4_75_(-if01)?-event-kbd'`